src/features/account/services/get-user-accounts-service.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, it, vi } from 'vitest';
+import { GetUserAccountsService } from './get-user-accounts-service.js';
+import { UserMock } from '@/shared/test-helpers/mocks/user.mock.js';
+
+const makeSut = () => {
+  const userRepository = {
+    create: vi.fn(),
+    findById: vi.fn(),
+  };
+
+  const accountRepository = {
+    getAccounts: vi.fn(),
+  };
+
+  const getUserAccountsService = new GetUserAccountsService(
+    userRepository,
+    accountRepository
+  );
+
+  return { accountRepository, getUserAccountsService, userRepository };
+};
+
+describe('GetUserAccountsService', () => {
+  it('should call userRepository.findById with correct id', async () => {
```

Comment:
evita o uso repetido de should nos seus testes, isso deixa eles pouco semânticos
```suggestion
  it('calls userRepository.findById with correct id', async () => {
```